src/directives/text-fit/text-fit.js

```diff
@@ -49,13 +49,13 @@ angular.module( 'gizmos.directives' ).value( 'textFit', function textFit( elemen
   // This is slow but WAY more reliable than el.scrollWidth. This method factors
   // in padding and such. Slower probably not an issue since the container is 
   // only computed once per font resize.
-  containerStyle = window.getComputedStyle(element.parent()[0])
-  if( containerStyle["box-sizing"] === 'border-box') {
-    containerWidth = parseInt(containerStyle.width, 10) - parseInt(containerStyle.paddingLeft, 10) - parseInt(containerStyle.paddingRight, 10) 
-    containerHeight = parseInt(containerStyle.height, 10)  - parseInt(containerStyle.paddingTop, 10) - parseInt(containerStyle.paddingBottom, 10)
+  containerStyle = window.getComputedStyle(element.parent()[0]);
+  if (containerStyle["box-sizing"] === "border-box") {
+    containerWidth = Math.round(parseFloat(containerStyle.width)) - Math.round(parseFloat(containerStyle.paddingLeft)) - Math.round(parseFloat(containerStyle.paddingRight));
+    containerHeight = Math.round(parseFloat(containerStyle.height)) - Math.round(parseFloat(containerStyle.paddingTop)) - Math.round(parseFloat(containerStyle.paddingBottom));
```

Comment:
Semicolons were used because I'm actually making my edits on the web-client-side (which has the transformed JS with semi-colons) to test and then I cut / pasted back into gizmos once I know it works. Just didn't see them. Math is updated.